stdlib/websub/src/main/ballerina/src/websub/Module.md

```diff
@@ -127,385 +104,26 @@ A hub client endpoint is also made available to publishers and subscribers to pe
-    runtime:sleep(5000);
-}
-```
-
-Ballerina publishers can also use the `websub:PublisherClient` to register topics at Ballerina WebSub hubs 
-and publish/notify updates to the remote hubs.
-```ballerina
-import ballerina/log;
-import ballerina/runtime;
-import ballerina/websub;
-
-websub:PublisherClient websubHubClientEP = new ("https://localhost:9191/websub/hub");
-
-public function main() {
-
-    var registrationResponse = websubHubClientEP->registerTopic("<TOPIC_URL>");
-    if (registrationResponse is error) {
-        log:printError("Error occurred registering topic: " + <string>registrationResponse.detail()?.message);
-    } else {
-        log:printInfo("Topic registration successful!");
-    }
-
-    // Make the publisher wait until the subscriber subscribes at the hub.
-    runtime:sleep(10000);
-
-    log:printInfo("Publishing update to remote Hub");
-    var publishResponse = websubHubClientEP->publishUpdate("<TOPIC_URL>", { "action": "publish", "mode": "remote-hub" });
-    if (publishResponse is error) {
-        log:printError("Error notifying hub: " + <string>publishResponse.detail()?.message);
-    } else {
-        log:printInfo("Update notification successful!");
-    }
-
-}
-```
-
-The `websub:SubscriptionClient` can be used by subscribers to send subscription and unsubscription requests explicitly.
-```ballerina
-import ballerina/log;
-import ballerina/websub;
-
-websub:SubscriptionClient websubHubClientEP = new("<HUB_URL>");
-
-public function main() {
-
-    // Send subscription request for a subscriber service.
-    websub:SubscriptionChangeRequest subscriptionRequest = {
-        topic: "<TOPIC_URL>", 
-        callback: "<CALLBACK_URL>",
-        secret: "<SECRET>"
-    };
-
-    var subscriptionChangeResponse = websubHubClientEP->subscribe(subscriptionRequest);
-    if (subscriptionChangeResponse is websub:SubscriptionChangeResponse) {
-        log:printInfo("Subscription Request successful at Hub [" + subscriptionChangeResponse.hub 
-                        + "] for Topic [" + subscriptionChangeResponse.topic + "]");
-    } else {
-        log:printError("Error occurred with Subscription Request", err = subscriptionChangeResponse);
-    }
-
-    // Send unsubscription request for the subscriber service.
-    websub:SubscriptionChangeRequest unsubscriptionRequest = {
-        topic: "<TOPIC_URL>",
-        callback: "<CALLBACK_URL>"
-    };
-
-    subscriptionChangeResponse = websubHubClientEP->unsubscribe(unsubscriptionRequest);
-    if (subscriptionChangeResponse is websub:SubscriptionChangeResponse) {
-        log:printInfo("Unsubscription Request successful at Hub [" + subscriptionChangeResponse.hub
-                + "] for Topic [" + subscriptionChangeResponse.topic + "]");
-    } else {
-        log:printError("Error occurred with Unsubscription Request", err = subscriptionChangeResponse);
-    }
-}
-```
-
-## Introducing Specific Subscriber Services (Webhook Callback Services)
```

Comment:
Shall we keep these details and replace the examples with code snippets and BBEs.